chapters/annotations.tex

```diff
@@ -888,6 +888,13 @@ \subsection{Component Instance}\label{component-instance}
 Sub-connectors in a hierarchical connector are only shown when they can be connected to.
 Non-connector components are only shown in the diagram layer.
 
+\subsubsection{Default Outline for Missing Graphics}\label{default-outline-for-missing-graphics}
+If the icon of a component placed in a diagram layer does not contain any graphical primitives (including inherited ones, and regardless of visibility; but excluding connectors), tools shall show a tool-dependent rudimentary outline of the component's transformed \lstinline!extent!.
```

Comment:
This is where I'm looking for a clear normative formulation instead of the vague term _visibility_.  Could it be this simple?
```suggestion
If the icon of a component placed in a diagram layer does not contain any graphical primitives (including inherited ones, and regardless of \lstinline!visible!-attributes; but excluding connectors), tools shall show a tool-dependent rudimentary outline of the component's transformed \lstinline!extent!.
```